### ISSUE TITLE
Compilation of fftwpp examples fails to find fftw and openmp

### ISSUE THREAD

**Issue by sbrisard** [sbrisard/fftwpp#2]
FFTW and openmp are dependencies of the ffwpp library and are specified as such in the CMake files. However, compilation of an example that specifies only fftwpp as a dependency fails. FFTW and OpenMP need to be specified explicitely.

**Comment by sbrisard** [comment]
Problem with FFTW is fixed in f61388002e520f3ef66709ec8662393ff6c6e307
```cpp
#include <fftw/fftw3.h>
```
replaced with 
```cpp
#include <fftw3.h>
```

**Comment by sbrisard** [comment]
Problem with OpenMP seems to be fixed in 5677d49b1bf5f125bc2df273420289f42a5a5aaa, where the `CONFIG` option was activated

```cmake
find_package(OpenMP CONFIG)
```
